src/main/java/seedu/address/logic/commands/DeleteStudentCommand.java

```diff
@@ -1,33 +1,65 @@
 package seedu.address.logic.commands;
 
+import static java.util.Objects.requireNonNull;
+
 import seedu.address.logic.commands.exceptions.CommandException;
 import seedu.address.model.Model;
-import seedu.address.model.student.StudentNumber;
+import seedu.address.model.person.Name;
+import seedu.address.model.student.Student;
 
 /**
- * Deletes a student from the system.
+ * Deletes a student from the system based on the student number provided.
  */
 public class DeleteStudentCommand extends Command {
     public static final String COMMAND_WORD = "deletestu";
 
     public static final String MESSAGE_USAGE = COMMAND_WORD
-            + ": Deletes the student identified by the student number used in the displayed student list.\n"
-            + "Parameters: STUDENT_NUMBER (must be a valid student number)\n"
-            + "Example: " + COMMAND_WORD + " A1234567B";
+            + ": Deletes the student identified by the student name used in the displayed student list.\n"
+            + "Parameters: STUDENT (must be a valid student name)\n"
+            + "Example: " + COMMAND_WORD + " Jerrell Lee";
+
+    public static final String MESSAGE_DELETE_STUDENT_SUCCESS = "Deleted Student: ";
```

Comment:
Should have %1$s for the string format?

Reply:
I've modified the logic to include the student name in the message sent.

Reply:
In the latest commit, he just used String concatenation to do it. It still works. 👍